packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -0,0 +1,94 @@
+import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import {
+  GoldTokenInstance,
+  RegistryInstance,
+  ReleaseGoldContract,
+  ReleaseGoldMultiSigContract,
+  ReleaseGoldMultiSigProxyContract,
+  ReleaseGoldProxyContract,
+} from 'types'
+import fs = require('fs')
+
+module.exports = async (callback: (error?: any) => number) => {
+  try {
+    const argv = require('minimist')(process.argv.slice(2), {
+      string: ['network', 'grants'],
+    })
+    const registry = await getDeployedProxiedContract<RegistryInstance>('Registry', artifacts)
+    const goldToken = await getDeployedProxiedContract<GoldTokenInstance>('GoldToken', artifacts)
+    const ReleaseGoldMultiSig: ReleaseGoldMultiSigContract = artifacts.require(
+      'ReleaseGoldMultiSig'
+    )
+    const ReleaseGoldMultiSigProxy: ReleaseGoldMultiSigProxyContract = artifacts.require(
+      'ReleaseGoldMultiSigProxy'
+    )
+    const ReleaseGold: ReleaseGoldContract = artifacts.require('ReleaseGold')
+    const ReleaseGoldProxy: ReleaseGoldProxyContract = artifacts.require('ReleaseGoldProxy')
+    const releases = []
+    const handleJSONFile = async (err, data) => {
```

Comment:
Are we doing no validation on the input? Seems kinda scary to me.

Reply:
Maybe we should add a prompt for each grant to confirm?

Reply:
+1 to that suggestion, though it can wait for a follow-on PR